Formula/archiver.rb

```diff
@@ -0,0 +1,42 @@
+class Archiver < Formula
+  desc "Cross-platform, multi-format archive utility"
+  homepage "https://github.com/mholt/archiver"
+  url "https://github.com/mholt/archiver/archive/v3.3.0.tar.gz"
+  sha256 "c8e88340e80b428c1a1c9734084395b473c9458fcea8b8b5126a9db96ae45844"
+  head "https://github.com/mholt/archiver.git"
+
+  depends_on "go" => :build
+
+  def install
+    # Don't set GOPATH because we want to build using go modules to
+    # ensure our dependencies are the ones specified in `go.mod`.
+    ENV["CGO_ENABLED"] = "0"
+    mkdir_p buildpath
+    ln_sf buildpath, buildpath/"archiver"
+
+    cd "archiver" do
+      system "go", "build", "-o", bin/"arc",
+             "cmd/arc/main.go"
+    end
```

Comment:
```suggestion
    system "go", "build", "-trimpath", "-o", bin/"arc", "cmd/arc/main.go"
```

Reply:
@dawidd6 Thank you for your suggestion! It is pretty much evident that I'm not fluent in go and this was in fact just c/p from `termshark` formula 🙈 
I would accept it straight away but shouldn't we keep cgo disable switch (line 13) due to: https://github.com/mholt/archiver/blob/44285f7/build.bash#L6 ?

Reply:
I don't think it's completely necessary, but okay, let's keep it if upstream does too.

Reply:
I was thinking the same. Kept it just for the sake of compatibility.

Reply:
It's probably for portability reasons. For example, if you compile against libc on Ubuntu and then want you use it on Alpine you might have issues, which `CGO_ENABLED=0` can avoid.

Reply:
Removed `CGO_ENABLED` var ✅